src/dotnet-interactive-vscode/src/vscode/notebookProvider.ts

```diff
@@ -34,7 +34,9 @@ export class DotNetInteractiveNotebookContentProvider implements vscode.Notebook
 
         let client = this.clientMapper.getOrAddClient(uri);
         let notebookPath = path.dirname(uri.fsPath);
-        client.changeWorkingDirectory(notebookPath);
+        client.changeWorkingDirectory(notebookPath).catch(() => {
+            vscode.window.showInformationMessage(`Unable to set notebook working directory to '${notebookPath}'.`);
```

Comment:
Should this information and message not be coming from the kernel in the form of a `CommandFailed`?

Reply:
Changed `reject()` to `reject(err)` and included the message.